easybuild/easyblocks/p/pgi.py

```diff
@@ -121,6 +121,13 @@ def install_step(self):
 
         cmd = "%s -x %s -g77 /" % (filename, install_abs_subdir)
         run_cmd(cmd, log_all=True, simple=True)
+        
+        # if an OS libnuma is NOT found, makelocalrc creates symbolic links to libpgnuma.so
+        # since we use the EB libnuma, delete those symbolic links
+        for filename in ["libnuma.so", "libnuma.so.1"]:
+            path = os.path.join(install_abs_subdir, "lib", filename)
+            if os.path.islink(path):
+                os.remove(path)
```

Comment:
@bartoldeman: you should only do this if `numactl` is actually included as a dependency?

use `if get_software_root('numactl')` to check